docs/ff-integrations/ads/admob.md

```diff
@@ -101,43 +84,50 @@ To add the APP keys:
+Below are some AdMob settings (under **Settings and Integrations** menu) that you might need to configure based on your app and target audience.
 
-# **AdBanner**
+- **Show Test Ads**: Test ads are placeholders provided by AdMob that simulate real ads. To enable test ads during development, enable this option. This allows you to click on ads without charging Google advertisers and prevents your account from being flagged for invalid activity. Once your app is ready for production, you can disable this setting to serve real ads.
 
-The AdBanner widget is used to show an advertisement banner inside your app. The AdBanner widget may display texts, images, and other rich media such as video ads. 
 
-:::info
-Our AdBanner widget uses Admob which is an Ad network from Google.
-:::
+- **Show GDPR Consent Dialog at App Launch**: To display the GDPR consent dialog for users in the European Union (EU), enable this option. **Note that** the dialog will only appear if the user is from the EU and you created a [European regulations message](https://support.google.com/admob/answer/10113207).
+- **Child-Directed Settings**: To indicate that your content is directed towards children, enable this option. This will ensure that Google treats your content as child-directed when making ad requests.
+- **Users Under the Age of Consent**: This setting allows you to comply with privacy regulations for users in the European Economic Area (EEA) who are under the age of consent. It ensures that ad requests are appropriately handled, limiting data collection and targeting to meet legal requirements. This is important to protect user privacy and to avoid penalties for non-compliance.
+- **Ad Content Filtering**: To filter the type of ads displayed, select the appropriate content rating. AdMob will ensure that ads returned for these requests have a content rating at or below the level selected.
+    
+    These are the levels you can set:
+    
+    - **G (General Audience)**: Suitable for all audiences, with no adult content or explicit themes.
+    - **PG (Parental Guidance)**: Ads may contain mild content, suitable for children with parental supervision.
+    - **T (Teen)**: Ads with content appropriate for teenagers; may include some mature topics.
+    - **MA (Mature Audience)**: Ads intended for adults, which may include strong themes or explicit content.
 
-Showing AdBanner From AdMob comprises of following steps:
+Once the setup is completed, you can start to display [AdBanner](#adbanner) or [Interstitial ads](#interstitial-ad) in your app.
 
-1. [Adding AdBanner widget](#1-adding-adbanner-widget)
-2. [Assigning Ad Unit ID](#2-assigning-ad-unit-id)
-3. [Enabling test ads](#3-enabling-test-ads)
-4. [Testing ads](#4-testing-ads)
+## AdBanner
 
-### 1. Adding AdBanner widget
+The **AdBanner** widget displays advertisement banners within your app. It can feature text, images, and rich media, including video ads.
```

Comment:
Any example image that includes adbanners in the ecommerce app?

Reply:
Yes, added one inside the tip admonition.